### ISSUE TITLE
missing closing braces

### ISSUE THREAD

**Issue by vhud** [dotnet/docs#21705]
The first code sample under the section 'Creating the first test' is missing a few closing braces. As the instructions are to replace the entire file's content with that code sample it's quite cruel to a newbie.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e0063ceb-57ea-f30c-efed-b2a36314ccaa
* Version Independent ID: bd9a321e-2ef2-491d-cd37-2d9c66f90cc9
* Content: [Unit testing C# with NUnit and .NET Core - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-nunit)
* Content Source: [docs/core/testing/unit-testing-with-nunit.md](https://github.com/dotnet/docs/blob/master/docs/core/testing/unit-testing-with-nunit.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @rprouse
* Microsoft Alias: **wiwagn**

**Comment by christianlevesque** [comment]
Is this issue available for newcomers? I've read the contributing docs and code of conduct.

**Comment by vhud** [comment]
I'm referring to the 'Creating the first test' section here: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-nunit

So yes anyone that looks at that page will see a code sample with missing closing braces. See my attached screenshot.

![image](https://user-images.githubusercontent.com/3059022/102302961-c4030380-3f1f-11eb-9fd2-2505141fa0bb.png)

**Comment by rprouse** [comment]
@christianlevesque this is an easy fix and a great place to jump in, so go for it. I am not commenting as someone who can merge your PR, but I wrote the original pages. This is a great opportunity to both give back and to learn the process.

**Comment by vhud** [comment]
Oh I totally misunderstood that question. If I have time later in the week I'll see if I can do it myself(unless @christianlevesque  beats me to it).

**Comment by christianlevesque** [comment]
@rprouse or @vhud, do either of you have any information on how to build the docs? I made my fix and I want to verify it works. I figured out a lot on my own but the `docfx` output only gives me image files, etc, nothing built. The first line in the build is:

```
Warning:[BuildCommand.ExportResourceFiles]No resource found for [docs.html]
```

and I assume that this is why none of the Markdown files get compiled, but I can't find any documentation on how `docs.html` might be constructed.

I'm also getting a lot of errors about invalid code snippet links. None of the directories seem to exist in the docs repo, such as `~/machinelearning-samples`, `~/_csharplang`, etc. I'm familiar with SSGs but this is the first time I've used `docfx`
